packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -1139,6 +1139,23 @@ new ecs.Ec2Service(this, 'EC2Service', {
 });
 ```
 
+### Cluster Default Provider Strategy
+
+When the service does not have a capacity provider strategy, the cluster's default capacity provider strategy will be used.
+In case of fargate service, if the cluster enableFargateCapacityProviders is false then default capacity provider strategy will be used.
+
+```ts
+declare const capacityProvider: ecs.CapacityProvider;
+
+new ecs.Cluster(stack, 'EC2CPCluster', {
+  vpc,
+  defaultCapacityProviderStrategy: [
```

Comment:
Can you help me understand why this is an array of values and not just a single value for the cluster?

Reply:
It's possible to create more complicated capacity providers strategies than just by specifying fargate as a string for the capacityprovider.